evaluation/src/main/java/com/formulasearchengine/mlp/evaluation/Evaluator.java

```diff
@@ -30,20 +30,40 @@
   private static final int IDENTIFIER = 2;
   private static final int DEFINITION = 3;
 
-  public Evaluator() {}
+  public Evaluator() {
+  }
 
   /**
-   * Evaluate an extraction result against the mlp gold standard.
+   * Evaluate an extraction result against the mlp gold standard. Uses the qId as key.
+   *
    * @param extractions multimap containing the identifiers and definitions for every formula.
-   * @param gold the gold data {@link #readGoldEntries}
+   * @param gold        the gold data {@link #readGoldEntries}
    * @return [true positives, false negatives, false positives]
    */
   public int[] evaluate(Multimap<String, IdentifierDefinition> extractions, List<GoldEntry> gold) {
+    return evaluate(extractions, gold, false);
+  }
+
+
+  /**
+   * Evaluate an extraction result against the mlp gold standard.
+   *
+   * @param extractions multimap containing the identifiers and definitions for every formula.
+   * @param gold        the gold data {@link #readGoldEntries}
+   * @param titleKey    if true the title will be used as key instead of the qId.
+   * @return [true positives, false negatives, false positives]
```

Comment:
I would prefere to return a custom datatype here. In the future it might be hard to remeber for other which value is at which position. Moreover one would need to handle exceptions where the int array is too short or too long